cpp/journal_timeliness_checker.cc

```diff
@@ -94,13 +94,8 @@ int Main(int argc, char *argv[]) {
 
         const std::string journal_name(section.getSectionName());
 
-        std::string journal_ppn(section.getString("online_ppn", ""));
-        if (journal_ppn.empty())
-            journal_ppn = section.getString("print_ppn", "");
-        if (journal_ppn.empty()) {
-            LOG_WARNING("no PPN found for \"" + journal_name + "\"!");
-            continue;
-        }
+        const std::string zeder_id(section.getString("zeder_id"));
+        const std::string zeder_instance(TextUtil::UTF8ToLower(section.getString("zotero_group")));
```

Comment:
zotero_group might also be RelBib, but i'm not sure if we should still allow that